x-pack/legacy/plugins/security/public/hacks/legacy.ts

```diff
@@ -0,0 +1,65 @@
+/*
```

Comment:
__note:__ decided to keep the remaining public legacy code in a single file like we did in the Spaces plugin.